src/main/java/com/lothrazar/cyclic/block/anvil/TileAnvilAuto.java

```diff
@@ -35,17 +35,26 @@
   static final int MAX = 64000;
   public static IntValue POWERCONF;
   CustomEnergyStorage energy = new CustomEnergyStorage(MAX, MAX);
-  private LazyOptional<IEnergyStorage> energyCap = LazyOptional.of(() -> energy);
+  private final LazyOptional<IEnergyStorage> energyCap = LazyOptional.of(() -> energy);
   ItemStackHandler inputSlots = new ItemStackHandler(1) {
+    @Override
+    public ItemStack insertItem(int slot, ItemStack stack, boolean simulate) {
+      if (!stack.isEmpty() && stack.isRepairable() && stack.getDamage() == 0) {
```

Comment:
Shifts repaired items arriving into input straight into output for use in automated lines.